client/src/components/shopInfo.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import style from './style.css.js';
+
+const ShopInfo = (props) => {
+  return (
+    <div className="shop" style={style.shop}>
+      <img src="http://loremflickr.com/75/75/tech?lock=1"></img>
```

Comment:
don't forget to swap these out for images from you s3

Reply:
Make sure you plan some time to learn how S3 works.  It'll take a couple hours.